Gopkg.toml

```diff
@@ -24,6 +24,9 @@
 #   go-tests = true
 #   unused-packages = true
 
+[[override]]
```

Comment:
what is this package for?

Reply:
When I run dep ensure, an error occurred.
https://github.com/golang/dep/issues/1799
https://github.com/cloudfoundry-incubator/disaster-recovery-acceptance-tests/commit/983620d3add93622b67c9d5abcd1abaa315c96c7

Reply:
is this signed off by lca?